Makefile

```diff
@@ -0,0 +1,135 @@
+all: fmt check
+
+# Always keep the last released version here
+VERSION ?= 0.0.1
+export VERSION := $(VERSION)
+
+TARGET_NAMESPACE ?= kubevirt-hyperconverged
+export TARGET_NAMESPACE := $(TARGET_NAMESPACE)
+
+DEPLOY_DIR ?= manifests
+
+# replace with quay.io/kubevirt once permissions are granted
+IMAGE_REGISTRY ?= quay.io/$(USER)
```

Comment:
It still would be good to have an option to override a user for development.

Reply:
when running `make docker-build` it will use $USER, in order apply a different user run with:
`QUAY_USER=other_user make build-docker push-docker`